pkg/miniogw/gateway-storj.go

```diff
@@ -72,48 +218,102 @@ func (s *storjObjects) GetObject(ctx context.Context, bucket, object string,
 }
 
 func (s *storjObjects) GetObjectInfo(ctx context.Context, bucket,
-	object string) (objInfo cmd.ObjectInfo, err error) {
+	object string) (objInfo minio.ObjectInfo, err error) {
 	panic("TODO")
 }
 
 func (s *storjObjects) ListBuckets(ctx context.Context) (
-	buckets []cmd.BucketInfo, err error) {
-	return []cmd.BucketInfo{{
-		Name:    "test-bucket",
-		Created: time.Now(),
-	}}, nil
+	buckets []minio.BucketInfo, err error) {
+	buckets, err = s.getbucketscallback()
```

Comment:
I don't see any benefit of extracting the logic in a private method. Perhaps, just move it back here inline.

Reply:
Then you can do it just like `return s.getbucketscallback()`.

Reply:
:-) I see... well another Misra coding style from the embedded system programming world....I see your point.. wilco ( will comply)